libs/core-ui/src/lib/Interfaces/IExplanationContext.ts

```diff
@@ -9,7 +9,25 @@ export enum ModelTypes {
   Regression = "regression",
   Binary = "binary",
   Multiclass = "multiclass",
-  Image = "image"
+  ImageMulticlass = "imagemulticlass",
+  TextMulticlass = "textmulticlass"
+}
+
+export function IsMulticlass(modelType: ModelTypes): boolean {
+  return (
+    modelType === ModelTypes.Multiclass ||
+    modelType === ModelTypes.ImageMulticlass ||
+    modelType === ModelTypes.TextMulticlass
+  );
+}
+
+export function IsClassifier(modelType: ModelTypes): boolean {
+  return (
+    modelType === ModelTypes.Binary ||
+    modelType === ModelTypes.Multiclass ||
+    modelType === ModelTypes.ImageMulticlass ||
+    modelType === ModelTypes.TextMulticlass
+  );
 }
```

Comment:
do you mind move these to util?
This interface folder suppose only have interface definition

Reply:
done!